cobalt/build/gn.py

```diff
@@ -173,4 +169,4 @@ def main(out_directory: str, platform: str, build_type: str,
     builds_out_directory = os.path.join(
         BUILDS_DIRECTORY, f'{script_args.platform}_{script_args.build_type}')
   main(builds_out_directory, script_args.platform, script_args.build_type,
-       script_args.overwrite_args, gen_args)
+       gen_args)
```

Comment:
You may already plan to do this (and don't need to do it here), but I would recommend getting rid of `gen_args` in favor of just having a check flag—i.e. limiting the script to not pass arbitrary args

Reply:
will address in https://github.com/youtube/cobalt/pull/4427

Reply:
👍